src/test/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderMatchesControllerIntTest.java

```diff
@@ -27,7 +28,7 @@
 @Sql(scripts = "classpath:after-test.sql", config = @SqlConfig(transactionMode = ISOLATED), executionPhase = AFTER_TEST_METHOD)
 public class OffenderMatchesControllerIntTest extends BaseIntTest {
 
-    private static final String COURT_CODE = "SHF";
+//    private static final String COURT_CODE = "SHF";
```

Comment:
This can go if it's not used

Reply:
Why did you decide to get rid of it as a matter of interest? I see it was used almost exclusively in String concatenations which does smell bit though I couldn't tell you why 😄

Reply:
Just created EntityHelper with a "real" looking court code and it filtered through to this - decided to standardise.